applications/h264/grpc_h264_endoscopy_tool_tracking/cpp/cloud/app_cloud_pipeline.hpp

```diff
@@ -97,15 +101,14 @@ class AppCloudPipeline : public HoloscanGrpcApplication {
         from_config("lstm_inference"),
         Arg("model_file_path", model_file_path),
         Arg("engine_cache_dir", engine_cache_dir),
-        Arg("pool") = make_resource<UnboundedAllocator>("pool"),
-        Arg("cuda_stream_pool") = make_resource<CudaStreamPool>("cuda_stream", 0, 0, 0, 1, 5));
+        Arg("pool") = make_resource<RMMAllocator>("pool"),
+        Arg("cuda_stream_pool") = cuda_stream_pool);
 
     auto tool_tracking_postprocessor = make_operator<ops::ToolTrackingPostprocessorOp>(
         "tool_tracking_postprocessor",
         streaming_enabled,
         from_config("tool_tracking_postprocessor"),
-        Arg("device_allocator") = make_resource<UnboundedAllocator>("device_allocator"),
-        Arg("host_allocator") = make_resource<UnboundedAllocator>("host_allocator"));
+        Arg("device_allocator") = make_resource<RMMAllocator>("device_allocator"));
```

Comment:
The ToolTrackingPostProcessorOp also has a cuda_stream_pool argument, please add the pool here as well.

Reply:
Thank you @AndreasHeumann

Reply:
The Edge app contains two fragments, and multi-fragment apps do not support serializing CUDA stream pools. They will give the following warning, so I have removed them from the Edge app.

```bash
[warning] [ucx_entity_serializer.cpp:199] No serializer found for component 'cuda_stream_id_' with type ID 0x7982aeac37f141beade86f00b4b5d47c
```

Is there any workaround?